mods/ts/rules/gdi-support.yaml

```diff
@@ -25,6 +25,26 @@ GAWALL:
 	LineBuild:
 		NodeTypes: wall, turret
 
+GAGATE_A:
+	Inherits: ^Gate_A
+	Buildable:
+		Queue: Defense
+		BuildPaletteOrder: 100
+		Prerequisites: gapile, ~structures.gdi
+	Tooltip:
+		Name: GDI Gate
+		Description: Stops infantry and blocks enemy fire.\nCan NOT be crushed by tanks.
```

Comment:
I don't think anybody would expect gates to be crushable by tanks?  Seems like redundant info to me.